services/ui/src/components/Breadcrumbs/index.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import Link from 'next/link';
+
+export default ({project, environment}) => {
```

Comment:
The `project` prop here is actually only the project name. In other places where `project` is used as a prop, an object is passed. The `environment` variable here is also an object. Changing this `projectName` I think would be better for understanding what's actually needed. (And of course, don't forget to change the prop names where this component is used).